### DISCUSSION TITLE
Do I need to add 8090 to docker-compose.yml to portforward qBittorrent?

### DISCUSSION THREAD

**Discussion by erickyun** [anasty17/mirror-leech-telegram-bot#1642] [Q&A]
Hi,

I would appreciate if someone could answer my question please?

**Comment by anasty17** [comment]
sure!

**Reply by erickyun** [reply]
Thank you for your response.

Can I ask why is bot not portforwarding qBittorrent by default? 
I'm asking because this needed feature it would boost speed.

So I need to add `- "8090:8090"` only to docker-compose.yml file right?

**Comment by anasty17** [comment]
How this could boost the speed ?

**Reply by erickyun** [reply]
"It helps to get a better ratio on private trackers, since you can't seed if you can't establish a p2p connection to the other peer (which is the case if they also have no port forwarding)."

"And if you only have a few seeders and they have no forwarding, you won't be able to download (same thing in reverse)."

"Port forwarding allows for seeding, faster seeding, and also allows other peers to find you, rather than your client having to go search for other peers."

**Comment by anasty17** [comment]
Webui only ui for what in docker u will use ur connection. U can control download or settings like management only!

**Reply by erickyun** [reply]
Then we need to port forward the Listening Port right? Then in qBittorrent.conf I need to add `Session\Port=49777` and then in docker-compose.yml file I need to enter `- "49777:49777"` correct?

**Comment by anasty17** [comment]
No keep it 8090 otherwise u need change in code

**Reply by erickyun** [reply]
Not webui port I did mean listening port

**Reply by anasty17** [reply]
Why u want to change this!

**Reply by anasty17** [reply]
u can open it but most of people don't prefer but who cares it's vps nothing dnager regarding security

**Reply by anasty17** [reply]
most of torrents using now dht and opening the port may lead to ban ur vps
this port for trackers
